### DISCUSSION TITLE
Should logo "do something"? (Or not?)

### DISCUSSION THREAD

**Discussion by mkmori** [processing/p5.js-web-editor#3140] [General]
It's a very minor, mostly academic thing, but I kind of expected the logo in the upper-left of the web editor to link to the homepage (or something)....

I'm not necessarily requesting or proposing a change, but for example, there doesn't seem to be a top-level feature on the web editor linking to the homepage. (Help menu options link to Reference and an "About" pop-up: The "About" pop-up does include a link to "Home".)

I'm unduly curious about thoughts or theories anyone may volunteer on appropriate/accessible functionality for this sort of marginal, "abstract" design element.... :)

**Comment by raclim** [comment]
Thanks for raising this! 

This has been mentioned a few times over the past few months, so I think it makes sense to have the logo redirect somewhere! There were a few suggestions on where this logo should do: point to p5js.org, refresh the editor page, or [link to the user's account or sketch list](https://github.com/processing/p5.js-web-editor/issues/3083). I'm not entirely sure what is the best option myself!

**Reply by mkmori** [reply]
Looks like it goes to homepage now, and I think that makes sense! :)
